cybereason.json

```diff
@@ -1576,7 +1817,7 @@
                 {
                     "data_path": "action_result.data.*.dns_query",
                     "data_type": "string",
-                    "column_name": "DNS Query",
+                    "column_name": "Dns Query",
```

Comment:
nit: I think the capital `DNS` is the correct version, right?

Reply:
Fixed